### PR TITLE
allow overriding cluster params from environment variables

### PR DESCRIPTION
**Pull request by nshidlin**

### CODE REVIEW

**Comment by lalon4** [comment]
reviewed again after changes.
/lgtm

File: discovery-infra/tests/base_test.py [hunk 1]
```diff
@@ -23,14 +23,22 @@ def node_controller(self, setup_node_controller):
         controller.format_all_node_disks()
 
     @staticmethod
-    def create_cluster(api_client):
+    def create_cluster(
+            api_client,
+            cluster_name=env_variables['CLUSTER_NAME'],
```
Comment (tsorya) [inline discovery-infra/tests/base_test.py]: we moved to lowercase

Comment (tsorya) [reply discovery-infra/tests/base_test.py]: CLUSTER_NAME -> cluster_name

Comment (nshidlin) [reply discovery-infra/tests/base_test.py]: done

**Pull request merged**